### DISCUSSION TITLE
Welcome to awesome-github-profiles Discussions!

### DISCUSSION THREAD

**Discussion by sanjay-kv** [recodehive/awesome-github-profiles#109] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sanjay-kv** [comment]
updated `gh-page` as main branch, so going further it will avoid confusion @HimangshuSharma01

**Comment by HimangshuSharma01** [comment]
> updated `gh-page` as main branch, so going further it will avoid confusion @HimangshuSharma01 

Okay sir
